src/unleash.ts

```diff
@@ -123,6 +126,17 @@ export class Unleash extends EventEmitter {
         tags,
         bootstrapProvider,
         bootstrapOverride,
+        eventSource:
+          experimentalMode?.type === 'streaming'
+            ? new EventSource(resolveUrl(unleashUrl, './client/streaming'), {
```

Comment:
we can expose the low level config options to the public API over time, but for now I set those defaults internally